packages/example-app-base/src/components/FeedbackList/FeedbackList.scss

```diff
@@ -0,0 +1,26 @@
+:global {
```

Comment:
I used this style for consistency because ExampleCard and ComponentPage within this same folder are doing it like that with the global classes. Should I still change it to local?

Reply:
Yeah, this should be changed to css modules.